src/main/java/com/spotify/sparkey/ByteBufferCleaner.java

```diff
@@ -85,9 +81,7 @@ private NewCleaner() throws ClassNotFoundException, NoSuchFieldException, Illega
     public void clean(MappedByteBuffer byteBuffer) {
       try {
         clean.invoke(theUnsafe, byteBuffer);
-      } catch (IllegalAccessException e) {
-        throw new RuntimeException(e);
-      } catch (InvocationTargetException e) {
+      } catch (IllegalAccessException | InvocationTargetException e) {
         throw new RuntimeException(e);
```

Comment:
Consider adding a message for context here (I know it's unrelated to this change)